PULL-REQUEST.md

```diff
@@ -0,0 +1,25 @@
+# Pull Request Template
+
+**Person who creates PR SHOULD NOT merge own pull request** 
```

Comment:
I would say don't use PR and pull request. Use one or the other. So in this case maybe it should say:

The person who creates the pull request SHOULD NOT merge their own pull request**

Reply:
✔️  Great catch. Changing this now